csharp/json/src/KafkaJsonDeserializer.cs

```diff
@@ -34,7 +34,7 @@ public KafkaJsonDeserializer(string schemaRegistryUrl, TokenCredential credentia
             {
                 Diagnostics =
                 {
-                    ApplicationId = "azsdk-net-KafkaJsonDeserializer/1.0"
```

Comment:
Is this change intentional? if so why do we need this?

Reply:
When testing I found that the serializer throws an error when the ApplicationId is longer than 25 characters